packages/contracts/scripts/deploy/manager.ts

```diff
@@ -1,137 +1,121 @@
-import { OptionDefinition } from "command-line-args";
+import simpleBondHandler from "./ubiquistick/SimpleBond";
+import { CMDType, Deploy_Manager_Type } from "../shared";
+import { createHandler } from "./create";
+import ubiquiStickSaleHandler from "./ubiquistick/UbiquiStickSale";
 
-import { DeployFuncCallback } from "../shared";
-import creditNFTFunc, { optionDefinitions as creditNFTOptions } from "./dollar/core/CreditNFT";
-import creditNFTManagerFunc, { optionDefinitions as creditNFTManagerOptions } from "./dollar/core/CreditNFTManager";
-import creditNFTRedemptionCalculatorFunc, { optionDefinitions as creditNFTRedemptionCalculatorOptions } from "./dollar/core/CreditNFTRedemptionCalculator";
-import creditRedemptionCalculatorFunc, { optionDefinitions as creditRedemptionCalculatorOptions } from "./dollar/core/CreditRedemptionCalculator";
-import dollarMintCalculatorFunc, { optionDefinitions as dollarMintCalculatorOptions } from "./dollar/core/DollarMintCalculator";
-import dollarMintExcessFunc, { optionDefinitions as dollarMintExcessOptions } from "./dollar/core/DollarMintExcess";
-import twapOracleDollar3poolFunc, { optionDefinitions as twapOracleDollar3poolOptions } from "./dollar/core/TWAPOracleDollar3pool";
-import ubiquityCreditTokenFunc, { optionDefinitions as ubiquityCreditTokenOptions } from "./dollar/core/UbiquityCreditToken";
-import ubiquityDollarManagerFunc, { optionDefinitions as ubiquityDollarManagerOptions } from "./dollar/core/UbiquityDollarManager";
-import ubiquityDollarTokenFunc, { optionDefinitions as ubiquityDollarTokenOptions } from "./dollar/core/UbiquityDollarToken";
-import ubiquityGovernanceTokenFunc, { optionDefinitions as ubiquityGovernanceTokenOptions } from "./dollar/core/UbiquityGovernanceToken";
+const dPath = "src/dollar";
+const cPath = "src/dollar/core";
+const uPath = "src/ubiquistick";
```

Comment:
```suggestion
const ubiquiStickPath = "src/ubiquistick";
```

Reply:
great suggestions, this will make them unambiguous.